srv/components/system/network/prepare.sls

```diff
@@ -19,6 +19,8 @@
 
 {% if nwmanager_is_running %}
 
+    {% set hostname_short = salt['cmd.run']('hostname -s') %}
```

Comment:
Should be `cmd.shell` if we are capturing the value.

Reply:
cmd.run for worked the same